### PR TITLE
Responsive for Mobile screen

### PR DESCRIPTION
**Pull request by edrisranjbar**
@abdolmajidbn worked on mobile screen for sign in and sign up pages. after a quick review; commits will be merged. but before that we have to test on different mobile screen to make sure it does not break at any point.

### CODE REVIEW

File: web/static/assets/style.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 :root {
+  --body_color: #f5f5f5;
```
Comment (edrisranjbar) [inline web/static/assets/style.css:2]: change the body_color variable name to **WhiteSmoke**

File: web/static/assets/style.css [hunk 2]
```diff
@@ -13,6 +14,7 @@
 body {
   font-family: Poppins;
   margin: 0px;
+  background-color: var(--body_color);
```
Comment (edrisranjbar) [inline web/static/assets/style.css:17]: user **WhiteSmoke** variable instead.

File: web/static/assets/style.css [hunk 3]
```diff
@@ -546,22 +547,79 @@ footer a {
     line-height: 18px;
   }
 
+  .sign_in_box {
```
Comment (edrisranjbar) [inline web/static/assets/style.css:550]: remove unnecessary lines.

File: web/static/assets/style.css [hunk 4]
```diff
@@ -546,22 +547,79 @@ footer a {
...
+
+  a.forgot_password,
+  a.have_account_link {
+    font-size: 12px;
+    font-weight: 400;
+  }
+  .success_button {
+    min-height: 30px;
+    min-width: 120px;
+    border-radius: 10px;
```
Comment (edrisranjbar) [inline web/static/assets/style.css:595]: remove border radius because its unnecessary.

**Review by edrisranjbar — COMMENTED** [review]
Totally its fine. but just remove some of the extra lines.

**Comment by abdolmajidbn** [comment]
Ok fine, On it.

**Comment by abdolmajidbn** [comment]
All done, just waiting for merge.

**Pull request merged**